Integrations/integration-Tanium.yml

```diff
@@ -189,21 +188,27 @@ script:
             parsed_param['Type'] = param['parameterType'].split('::')[-1] if param.get('parameterType') else 'Type not specified'
             parsed_params.append(parsed_param)
             parsed_param = {}
-        return tableToMarkdown('Package Arguments Details', parsed_params, ['Key','Name', 'Values', 'Description', 'Type'])
+        return tableToMarkdown(header, parsed_params, ['Key','Name', 'Values', 'Description', 'Type'])
 
     def getPackage(handler):
         response = getObject(handler, u'package')
-        parsed = response['package_spec']
+        parsed = response.get('package_spec')
         final_result = create_entry('Tanium Package', parsed, {'Tanium.Packages' : parsed}, ['id', 'name', 'creation_time', 'command', 'last_modified_by'])
-        final_result['HumanReadable'] += parameterTableBuilder(parsed[0])
+        final_result['HumanReadable'] += parameterTableBuilder(parsed[0], 'Package Arguments Details')
         return final_result
 
     def getSavedQuestion(handler):
         response = getObject(handler, u'saved_question')
-        parsed = response['saved_question']
+        parsed = response.get('saved_question')
         final_result = create_entry('Tanium Saved Question', parsed, {'Tanium.SavedQuestions' : parsed}, ['query_text', 'name', 'id'])
         return final_result
 
+    def getSensor(handler):
```

Comment:
![image](https://user-images.githubusercontent.com/18641362/44514698-7c5e6b00-a6c9-11e8-863a-ec66f0981b0b.png)
not sure why isn't `tn-get-sensor` is not working

Reply:
are you on demo2? because I'm working on tanium_copy there @Bars92

Reply:
yeah, ill check locally

Reply:
works good locally !